apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -128,6 +128,9 @@ export default class AnimationPickerBody extends React.Component {
   animationCategoriesRendering() {
     const libraryManifest = this.props.getLibraryManifest();
     const categories = libraryManifest.categories;
+    // Add "All" category, which isn't in the manifest categories,
+    // but with an empty list since searchAssets handles All separately.
```

Comment:
nit: It feels clearer to me to do something like `const categories = Object.keys(libraryManifest.categories) + ["all"]` because we don't use the actual values of `categories` if I'm reading the code correctly.